### PR TITLE
spotify: Add version 1.1.80.699.gc3dac750

### LINKED ISSUE #7962: [Request] Spotify
### Information

**Name:** Spotify
**Description:** Digital music service.

**Homepage:** https://www.spotify.com/

**Download link(s):** https://download.scdn.co/SpotifyFullSetup.exe

**Some indication of popularity/repute:**

### PR DESCRIPTION
**Pull request by sitiom**
Closes #7962

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/spotify.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    ],
+    "uninstaller": {
+        "script": "Start-Process -Wait \"$dir\\Spotify.exe\" -ArgumentList '/Uninstall', '/Silent'"
+    },
+    "checkver": {
+        "script": [
+            "$download_url = 'https://download.scdn.co/SpotifyFullSetup.exe'",
+            "$download = cache_path 'spotify' 'unknown' $download_url",
+            "do_dl $download_url $download",
+            "$version = (Get-Item $download).VersionInfo.ProductVersion",
```
Comment (rashil2000) [inline bucket/spotify.json:42]: Use `spotify_version` instead of `version`

File: bucket/spotify.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        "identifier": "Freeware",
+        "url": "https://www.spotify.com/legal/end-user-agreement/"
+    },
+    "url": "https://download.scdn.co/SpotifyFullSetup.exe",
+    "hash": "33bab2581b9d94bb5c4a652266deedf5b09e420f3a81cd6cc004194c1d845836",
+    "installer": {
+        "script": [
+            "Start-Process -Wait \"$dir\\$fname\" -ArgumentList '/extract', \"`\"$dir`\"\"",
+            "Remove-Item \"$dir\\$fname\"",
+            "",
```
Comment (rashil2000) [inline bucket/spotify.json:15]:
```suggestion
            "Remove-Item \"$dir\\$fname\", \"$dir\\SpotifyMigrator.exe\"",
```

File: bucket/spotify.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            "# Disable built-in updater",
+            "Remove-Item -ErrorAction Ignore -Recurse \"$env:LOCALAPPDATA\\Spotify\\Update\" | Out-Null",
+            "$updateFile = New-Item -Path \"$env:LOCALAPPDATA\\Spotify\" -Name Update -ItemType File -Value \"Disabled by Scoop\" -Force",
+            "$updatefile.Attributes = 'ReadOnly', 'System'",
+            "Remove-Item \"$dir\\SpotifyMigrator.exe\""
```
Comment (rashil2000) [inline bucket/spotify.json:20]:
```suggestion
            "$updatefile.Attributes = 'ReadOnly', 'System'"
```

**Review by rashil2000 — COMMENTED** [review]
Shouldn't you delete the file created by Scoop in AppData, during uninstallation?

**Comment by sitiom** [comment]
> Shouldn't you delete the file created by Scoop in AppData, during uninstallation?

The `uninstaller` already does that (It deletes the entire `Spotify` folder in `%localappdata%`).

**Pull request merged**